main.js

```diff
@@ -0,0 +1,133 @@
+let currentBoard = { 
+  }
+
+function newGame(){
+  currentBoard[0]= [5,4,3,2,1],
+  currentBoard[1]= [];
+  currentBoard[2]=[];
+  return(currentBoard)
+}
+function showBoard(){
+  console.log(Object.entries(currentBoard).map((i) => {return("---" + i[1].join(" "))}))
+}
+
+newGame()
+
+function moveDisc(disc, peg){
+  // let lastPeg = peg[peg.length-1];
+  let lastDiscPeg1 = currentBoard[0][currentBoard[0].length-1];
+  let lastDiscPeg2 = currentBoard[1][currentBoard[1].length-1];
+  let lastDiscPeg3 = currentBoard[2][currentBoard[2].length-1];
+  let largeOnSmall = "Sorry, Charlie, you can't place larger discs on top of smaller discs. Try again!";
+  let notLastDisc = "Sorry, Charlie, you can only move the last disc on a peg! Try again"
+  if (peg === 1){
+    peg = currentBoard[0];
+    if (peg[peg.length-1] > disc || peg.length === 0){
+      // peg.push(disc);
+      if (currentBoard[1].includes(disc)){
+        if (lastDiscPeg2 === disc)
+        {currentBoard[1].pop(disc);
```

Comment:
bad use of pop, it doesnt require an argument, always removes the last item on the list